src/Datasource/Paginator.php

```diff
@@ -220,7 +220,10 @@ public function paginate($object, array $params = [], array $settings = [])
         $this->_pagingParams = [$alias => $paging];
 
         if ($requestedPage > $page) {
-            throw new PageOutOfBoundsException([$requestedPage]);
+            throw new PageOutOfBoundsException([
+                'requestedPage' => $requestedPage,
+                'pagingParams' => $this->_pagingParams
```

Comment:
Given that `$this->_pagingParams` is reset for every call to `paginate()` I don't see how it could have multiple aliases/keys.